From: Tommy Webb <tommy@calyxinstitute.org>
Date: Thu, 15 Feb 2024 20:10:16 +0000
Subject: Update expectations

There was an error or warning when building that suggested doing this.
The updating was done using a patch command that was given as part of
the error or warning message.

More info here (link came along with error):
https://chromium.googlesource.com/chromium/src/+/HEAD/chrome/android/expectations/README.md

Change-Id: I53210b52437c0563c054942a3b024ae8392c3cf1
---
 ...4_32_bundle__base.AndroidManifest.expected | 51 +++++++------------
 ...y_64_32_apk.arm64.libs_and_assets.expected |  1 -
 2 files changed, 17 insertions(+), 35 deletions(-)

diff --git a/chrome/android/expectations/trichrome_chrome_64_32_bundle__base.AndroidManifest.expected b/chrome/android/expectations/trichrome_chrome_64_32_bundle__base.AndroidManifest.expected
--- a/chrome/android/expectations/trichrome_chrome_64_32_bundle__base.AndroidManifest.expected
+++ b/chrome/android/expectations/trichrome_chrome_64_32_bundle__base.AndroidManifest.expected
@@ -1,7 +1,7 @@
 <?xml version="1.0" ?>
 <manifest
     xmlns:android="http://schemas.android.com/apk/res/android"
-    package="org.chromium.chrome.stable"
+    package="org.chromium.chrome"
     platformBuildVersionCode="35"
     platformBuildVersionName="15"
     android:isolatedSplits="true"
@@ -12,21 +12,12 @@
   <permission android:name="$PACKAGE.permission.CHILD_SERVICE" android:protectionLevel="signature"/>
   <permission android:name="$PACKAGE.permission.DEBUG" android:label="Debug web pages" android:protectionLevel="signature"/>
   <permission android:name="$PACKAGE.permission.READ_WRITE_BOOKMARK_FOLDERS" android:protectionLevel="signatureOrSystem"/>
-  <queries>  # DIFF-ANCHOR: 9588fea7
-    <package android:name="com.google.ar.core"/>
-  </queries>  # DIFF-ANCHOR: 9588fea7
   <uses-feature android:glEsVersion="0x00020000"/>
   <uses-feature android:name="android.hardware.camera" android:required="false"/>
   <uses-feature android:name="android.hardware.camera.autofocus" android:required="false"/>
   <uses-feature android:name="android.hardware.location.gps" android:required="false"/>
   <uses-feature android:name="android.hardware.microphone" android:required="false"/>
-  <uses-feature android:name="android.hardware.screen.landscape" android:required="false"/>
-  <uses-feature android:name="android.hardware.sensor.accelerometer" android:required="false"/>
-  <uses-feature android:name="android.hardware.sensor.gyroscope" android:required="false"/>
   <uses-feature android:name="android.hardware.touchscreen" android:required="false"/>
-  <uses-feature android:name="android.hardware.vr.headtracking" android:required="false" android:version="1"/>
-  <uses-feature android:name="android.hardware.vr.high_performance" android:required="false"/>
-  <uses-feature android:name="android.software.vr.mode" android:required="false"/>
   <uses-permission android:name="$PACKAGE.TOS_ACKED"/>
   <uses-permission android:name="$PACKAGE.permission.C2D_MESSAGE"/>
   <uses-permission android:name="$PACKAGE.permission.READ_WRITE_BOOKMARK_FOLDERS"/>
@@ -60,29 +51,25 @@
   <uses-permission android:name="com.chrome.permission.DEVICE_EXTRAS"/>
   <uses-permission android:name="com.google.android.apps.now.CURRENT_ACCOUNT_ACCESS"/>
   <uses-permission android:name="com.google.android.c2dm.permission.RECEIVE"/>
-  <uses-permission-sdk-23 android:name="android.permission.ACCESS_ADSERVICES_ATTRIBUTION"/>
   <uses-permission-sdk-23 android:name="android.permission.ACCESS_WIFI_STATE"/>
   <uses-permission-sdk-23 android:name="android.permission.BLUETOOTH"/>
   <uses-permission-sdk-23 android:name="android.permission.BLUETOOTH_ADMIN" android:maxSdkVersion="30"/>
   <uses-permission-sdk-23 android:name="android.permission.BLUETOOTH_ADVERTISE"/>
   <uses-permission-sdk-23 android:name="android.permission.BLUETOOTH_CONNECT"/>
   <uses-permission-sdk-23 android:name="android.permission.BLUETOOTH_SCAN" android:usesPermissionFlags="neverForLocation"/>
-  <uses-permission-sdk-23 android:name="android.permission.HAND_TRACKING"/>
-  <uses-permission-sdk-23 android:name="android.permission.READ_CONTACTS"/>
   <uses-permission-sdk-23 android:name="android.permission.READ_MEDIA_AUDIO"/>
   <uses-permission-sdk-23 android:name="android.permission.READ_MEDIA_IMAGES"/>
   <uses-permission-sdk-23 android:name="android.permission.READ_MEDIA_VIDEO"/>
   <uses-permission-sdk-23 android:name="android.permission.REORDER_TASKS"/>
   <uses-permission-sdk-23 android:name="android.permission.REQUEST_INSTALL_PACKAGES"/>
-  <uses-permission-sdk-23 android:name="android.permission.SCENE_UNDERSTANDING_FINE"/>
   <uses-permission-sdk-23 android:name="android.permission.USE_BIOMETRIC"/>
   <uses-permission-sdk-23 android:name="android.permission.USE_FINGERPRINT"/>
   <uses-sdk android:minSdkVersion="29" android:targetSdkVersion="35"/>
   <application
-      android:name="org.chromium.chrome.browser.base.SplitChromeApplication"
+      android:name="$PACKAGE.browser.base.SplitChromeApplication"
       android:allowAudioPlaybackCapture="false"
       android:allowBackup="false"
-      android:appComponentFactory="org.chromium.chrome.browser.base.SplitCompatAppComponentFactory"
+      android:appComponentFactory="$PACKAGE.browser.base.SplitCompatAppComponentFactory"
       android:enableOnBackInvokedCallback="true"
       android:extractNativeLibs="false"
       android:icon="@drawable/ic_launcher"
@@ -94,7 +81,7 @@
       android:networkSecurityConfig="@xml/network_security_config"
       android:roundIcon="@drawable/ic_launcher_round"
       android:supportsRtl="true"
-      android:zygotePreloadName="org.chromium.chrome.app.TrichromeZygotePreload">
+      android:zygotePreloadName="$PACKAGE.app.TrichromeZygotePreload">
     <activity  # DIFF-ANCHOR: ea1a94af
         android:name="com.google.android.gms.common.api.GoogleApiActivity"
         android:exported="false"
@@ -110,10 +97,6 @@
     <meta-data android:name="android.content.APP_RESTRICTIONS" android:resource="@xml/app_restrictions"/>
     <meta-data android:name="com.google.android.gms.cast.framework.OPTIONS_PROVIDER_CLASS_NAME" android:value="org.chromium.components.media_router.caf.CastOptionsProvider"/>
     <meta-data android:name="com.google.android.gms.version" android:value="@integer/google_play_services_version"/>
-    <meta-data android:name="com.google.ar.core" android:value="optional"/>
-    <meta-data android:name="com.google.ar.core.load_32bit_so_from_split_module" android:value="optional"/>
-    <meta-data android:name="com.google.ar.core.min_apk_version" android:value="240920000"/>
-    <meta-data android:name="com.google.ar.core.session_settings" android:value="allow_front_facing_motion_tracking_6dof,"/>
     <meta-data android:name="com.samsung.android.sdk.multiwindow.enable" android:value="true"/>
     <meta-data android:name="com.samsung.android.sdk.multiwindow.penwindow.enable" android:value="true"/>
     <meta-data android:name="org.chromium.content.browser.NUM_PRIVILEGED_SERVICES" android:value="10"/>
@@ -123,38 +106,38 @@
         android:authorities="$PACKAGE.AutofillThirdPartyModeContentProvider"
         android:exported="true">
     </provider>  # DIFF-ANCHOR: 3b9c6c31
-    <provider  # DIFF-ANCHOR: b73a881a
-        android:name="org.chromium.chrome.browser.download.DownloadFileProvider"
+    <provider  # DIFF-ANCHOR: f0a9ffaa
+        android:name="$PACKAGE.browser.download.DownloadFileProvider"
         android:authorities="$PACKAGE.DownloadFileProvider"
         android:exported="false"
         android:grantUriPermissions="true">
       <meta-data android:name="android.support.FILE_PROVIDER_PATHS" android:resource="@xml/file_paths"/>
-    </provider>  # DIFF-ANCHOR: b73a881a
-    <provider  # DIFF-ANCHOR: 43f90318
-        android:name="org.chromium.chrome.browser.pdf.PdfContentProvider"
+    </provider>  # DIFF-ANCHOR: f0a9ffaa
+    <provider  # DIFF-ANCHOR: 091f3ba6
+        android:name="$PACKAGE.browser.pdf.PdfContentProvider"
         android:authorities="$PACKAGE.PdfContentProvider"
         android:exported="false"
         android:grantUriPermissions="true">
-    </provider>  # DIFF-ANCHOR: 43f90318
-    <provider  # DIFF-ANCHOR: f1c2d7e1
-        android:name="org.chromium.chrome.browser.provider.ChromeBrowserProvider"
+    </provider>  # DIFF-ANCHOR: 091f3ba6
+    <provider  # DIFF-ANCHOR: 37f48914
+        android:name="$PACKAGE.browser.provider.ChromeBrowserProvider"
         android:authorities="$PACKAGE.ChromeBrowserProvider;$PACKAGE.browser;$PACKAGE"
         android:exported="true">
       <path-permission android:path="/bookmarks/search_suggest_query" android:readPermission="android.permission.GLOBAL_SEARCH"/>
-    </provider>  # DIFF-ANCHOR: f1c2d7e1
+    </provider>  # DIFF-ANCHOR: 37f48914
     <provider  # DIFF-ANCHOR: 6e040436
         android:name="org.chromium.chrome.browser.provider.PageContentProvider"
         android:authorities="$PACKAGE.PageContentProvider"
         android:exported="false"
         android:grantUriPermissions="true">
     </provider>  # DIFF-ANCHOR: 6e040436
-    <provider  # DIFF-ANCHOR: 67014fab
-        android:name="org.chromium.chrome.browser.util.ChromeFileProvider"
+    <provider  # DIFF-ANCHOR: 8845e3f4
+        android:name="$PACKAGE.browser.util.ChromeFileProvider"
         android:authorities="$PACKAGE.FileProvider"
         android:exported="false"
         android:grantUriPermissions="true">
       <meta-data android:name="android.support.FILE_PROVIDER_PATHS" android:resource="@xml/file_paths"/>
-    </provider>  # DIFF-ANCHOR: 67014fab
+    </provider>  # DIFF-ANCHOR: 8845e3f4
     <provider  # DIFF-ANCHOR: cb51666f
         android:name="org.chromium.ui.dragdrop.DropDataContentProvider"
         android:authorities="$PACKAGE.DropDataProvider"
@@ -559,6 +542,6 @@
         android:process=":sandboxed_process9">
     </service>  # DIFF-ANCHOR: e2f3bbbd
     <uses-native-library android:name="libopenxr.google.so" android:required="false"/>
-    <uses-static-library android:name="org.chromium.trichromelibrary" android:certDigest="32a2fc74d731105859e5a85df16d95f102d85b22099b8064c5d8915c61dad1e0" android:version="OFFSET=33"/>
+    <uses-static-library android:name="org.chromium.trichromelibrary" android:certDigest="d674c702c9415fb3b6ee9a1a80b8c45ad6ea5395379eb21f51122938c15f6be8" android:version="OFFSET=33"/>
   </application>
 </manifest>
diff --git a/chrome/android/expectations/trichrome_library_64_32_apk.arm64.libs_and_assets.expected b/chrome/android/expectations/trichrome_library_64_32_apk.arm64.libs_and_assets.expected
--- a/chrome/android/expectations/trichrome_library_64_32_apk.arm64.libs_and_assets.expected
+++ b/chrome/android/expectations/trichrome_library_64_32_apk.arm64.libs_and_assets.expected
@@ -1,4 +1,3 @@
-apk_path=lib/arm64-v8a/libarcore_sdk_c.so, compress=False, alignment=16384
 apk_path=lib/arm64-v8a/libcrashpad_handler_trampoline.so, compress=False, alignment=16384
 apk_path=lib/arm64-v8a/libmonochrome_64.so, compress=False, alignment=16384
 apk_path=assets/icudtl.dat+com.android.webview+, compress=False, alignment=4
-- 

